askama_shared/src/generator.rs

```diff
@@ -1576,6 +1642,30 @@ where
     }
 }
 
+impl MapChain<'_, &str, Option<String>> {
+    /// Given a variable `key` it resolves all the way
+    /// back to the initial the variable.
+    fn resolve_var<'a>(&'a self, key: &'a str) -> Option<&'a str> {
```

Comment:
What's the argument against just calling this `resolve()`?

Reply:
Originally, `resolve_var` was a helper function, so I kinda just kept the name. But I'll rename it to `resolve()` :)